modules/dcache/src/main/resources/diskCacheV111/services/transfer-manager.xml

```diff
@@ -132,7 +132,6 @@
         <property name="moverTimeoutUnit" value="${transfermanagers.limits.transfer-time.unit}"/>
         <property name="maxTransfers" value="${transfermanagers.limits.external-transfers}"/>
         <property name="ioQueueName" value="${transfermanagers.mover.queue}"/>
-        <property name="persistenceManagerFactory" ref="pmf"/>
```

Comment:
Probably you need to remove the initialization of `pmf` bean too

Reply:
the `pmf` still there...

Reply:
I've removed it --  not sure if this snippet will show the changes but if you go to the Files Changed tab you'll see the change